src/dtls/mod.rs

```diff
@@ -113,8 +114,8 @@ pub enum DtlsEvent {
     /// When the DTLS has finished handshaking.
     Connected,
 
-    /// Keying material for SRTP encryption master key.
-    SrtpKeyingMaterial(KeyingMaterial),
+    /// Keying material for SRTP encryption master key and the selected SRTP profile.
+    SrtpKeyingMaterial(KeyingMaterial, SrtpProfile),
```

Comment:
In addition to the keying material from the DTLS session we also need to bubble up the selected SRTP profile since we now negotiate more than one